ppb/systems/sound.py

```diff
@@ -141,14 +142,18 @@ def on_play_sound(self, event, signal):
         sound = event.sound
         chunk = event.sound.load()
 
-        channel = _call(
-            Mix_PlayChannel,
-            -1,  # Auto-pick channel
-            chunk,
-            0,  # Do not repeat
-            _check_error=lambda rv: rv == -1
-        )
-        self._currently_playing[channel] = sound  # Keep reference of playing asset
+        try:
+            channel = _call(
+                Mix_PlayChannel,
+                -1,  # Auto-pick channel
+                chunk,
+                0,  # Do not repeat
+                _check_error=lambda rv: rv == -1
+            )
+            self._currently_playing[channel] = sound  # Keep reference of playing asset
```

Comment:
Can we move this to an `else` of the `try`? Just because I generally like to keep the `try` bodies as minimal as possible.

Reply:
I literally had that thought after I submitted it.